spec/factory_bot/attribute_spec.rb

```diff
@@ -1,7 +1,15 @@
 describe FactoryBot::Attribute do
-  let(:name)  { "user" }
-  subject     { FactoryBot::Attribute.new(name, false) }
+  it "should convert the name attribute to a symbol" do
```

Comment:
I don't have strong feelings about this at all, but I think it is fun that somebody was opinionated enough about removing the word "should" that they created a gem for it: https://github.com/should-not/should_not

Reply:
😆